scripts/infra/local-setup.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash -e
+# SPDX-License-Identifier: Apache-2.0
+
+ensure_aws_system_pkgs() {
```

Comment:
This should be rh__ensure-aws-system-pkgs so it hits the first block in rh()

Reply:
Done!